docs/experimental-organization.md

```diff
@@ -711,3 +731,30 @@ At the current stage of Phis development, datasets cannot be modified nor delete
 ### Visualize Dataset
 Graphics extracted from Phis datasets are available through the `Experiments` menu.
 See the [Map Visualization](../experimental-organization/#graphics-from-datasets) section of this documentation for further information.
+
+## Annotations
+
+### What are annotations ?
+
+Phis project, experiments, sensors and vectors can be annotated using the W3C specifications of the [Web Annotation Data model](https://www.w3.org/TR/annotation-model/), or `oa` (prefix of the Web Annotation Data Model, whose namespace is [http://www.w3.org/ns/oa#](http://www.w3.org/ns/oa#)).
+Any user can therefore enrich Phis elements with attached information using semantic annotations.
```

Comment:
You could use "metadata"

Reply:
instead of or in addition to "Phis elements with attached information using semantic annotations" ?

Reply:
Just use it in one place in the text, because it's a keeword. Maybe in the first paragraph of annotation..

Reply:
correction made with the commit 2121441d3a5b35cd3cda86fe9d6553fb055e9cbb